### DISCUSSION TITLE
How do I deploy a Reach Contract to Algorand Test net

### DISCUSSION THREAD

**Discussion by BdVade** [reach-sh/reach-lang#1066] [Q&A] [answered]
I have a contract and it's front end. I'm not sure what to do to deploy it to the Algorand Test net
Here is what I have so far.
https://github.com/Adenikym/Auction-app/blob/86b509ed4e3190d1832458c853fe120330db1770/src/redux/actions/action.js#L152

**Comment by jeapostrophe** [comment, marked as answer]
You just attach to TestNet and run your code, and now it's own TestNet. Reach doesn't care what network you're on... you can just pick it and the rest of the code just works --- https://docs.reach.sh/frontend/#ref-frontends-js-provider
